config/kubernetes-sigs/sig-aws/teams.yaml

```diff
@@ -12,6 +12,7 @@ teams:
     - d-nishi
     - jsafrane
     - leakingtapan
+    - gnufied
```

Comment:
This needs to be alphabetically sorted and you'll also need to add yourself as a @kubernetes-sigs member here - https://github.com/kubernetes/org/blob/master/config/kubernetes-sigs/org.yaml